src/modules/services_manager_service.rb

```diff
@@ -59,39 +105,35 @@ def read
 
       private
 
+      # FIXME: use Yast::Systemctl for this, remember to chomp SERVICE_SUFFIX
+
+      # @return [Array<String>] "apache2.service   enabled\n"
       def list_unit_files
         command = TERM_OPTIONS + LIST_UNIT_FILES_COMMAND + COMMAND_OPTIONS
-        SCR.Execute(Path.new('.target.bash_output'), command)
+        out = SCR.Execute(Path.new('.target.bash_output'), command)['stdout']
+        out.lines
       end
 
+      # @return [Array<String>] "dbus.service   loaded active running D-Bus System Message Bus\n"
       def list_units
         command = TERM_OPTIONS + LIST_UNITS_COMMAND + COMMAND_OPTIONS
-        SCR.Execute(Path.new('.target.bash_output'), command)
-      end
-
-      # Checking if a service is active or not.
-      #
-      # @param service [String] Service name
-      # @return [Boolean] is it active or not
-      def is_active?(service)
-        # There is a active? method in SystemdUnit class but it checks the status
-        # active and activating only. Not sure if this correct. So we are taking the
-        # official call of systemctl command.
-        command = "#{TERM_OPTIONS}#{IS_ACTIVE_COMMAND} #{service}#{SERVICE_SUFFIX} 2>&1"
-        SCR.Execute(Path.new('.target.bash_output'), command)["exit"] == 0
+        out = SCR.Execute(Path.new('.target.bash_output'), command)['stdout']
+        out.lines
       end
 
       def load_unit_files
-        list_unit_files['stdout'].each_line do |line|
+        list_unit_files.each do |line|
           service, status = line.split(/[\s]+/)
           service.chomp! SERVICE_SUFFIX
+          # Unit template, errors out when inquired with `systemctl show`
```

Comment:
any reference where to get more info about it?

Reply:
Added